src/backend/context.rs

```diff
@@ -406,40 +282,44 @@ impl Context {
 		event_loop: &EventLoopWindowTarget,
```

Comment:
Not specific to this line. 

I am curious why the structure is the following:
impl Context {}
impl ContextHandle {}
impl Context{}

Why not have 
impl Context { `every function here`}
impl ContextHandle{}

Reply:
Hmm, this is a relic from back when I still planned to expose `Context` directly to users of the crate. I put all the public APIs at the top and then the internal stuff at the buttom. But now `Context` itself is private, so it doesn't make much sense anymore.

I'll move it around after this PR :)